### ISSUE TITLE
Support PositiveOrZero annotation from javax.validation

### ISSUE THREAD

**Issue by klDen** [mtedone/podam#283]
Hello,

Thank you for this nice lib!

Using the annotation **@PositiveOrZero** returns an error. Can you please support this annotation?

Currently, I'm bypassing the error with
`        ((RandomDataProviderStrategy)podamFactory.getStrategy()).addOrReplaceAttributeStrategy(PositiveOrZero.class,
        (AttributeStrategy<Number>) (attrType, attrAnnotations) -> 10000L);`

Used podam version: **7.2.4.RELEASE**
Used hibernate validation version: **6.1.5.Final** 

Thank you!

**Comment by klDen** [comment]
Please support @Positive as well. I am currently using
```
    ((RandomDataProviderStrategy) PODAM_FACTORY.getStrategy())
        .addOrReplaceAttributeStrategy(
            Positive.class,
            (AttributeStrategy<Number>)
                (attrType, attrAnnotations) -> FAKER.number().numberBetween(1, Long.MAX_VALUE));
```

**Comment by daivanov** [comment]
Fixed in podam 7.2.5
